### PR TITLE
Full text search lunr

### PR DESCRIPTION
**Pull request by Stephenitis**
@yaron check this out and see for functionality

### CODE REVIEW

**Comment by yaron** [comment]
I am guessing you mean @ysadka ?

**Comment by Stephenitis** [comment]
sorry @yaron tai! lol github autolinking fail.

On Tue, Aug 19, 2014 at 2:18 AM, Yaron Tal notifications@github.com wrote:

> I am guessing you mean @ysadka https://github.com/ysadka ?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/iron-io/docs/pull/414#issuecomment-52609035.

## 

Stephen Nguyen,
Developer Evangelist, Iron.io http://www.iron.io/

My General Availability https://ironpairing.appointlet.com/

http://www.linkedin.com/in/stephenhnguyen/
http://www.github.com/stephenitis  http://www.twitter.com/stephenitis

File: _plugins/index_all_pages.rb [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+end
+module Jekyll
+
+  class SearchIndexFile < StaticFile
+    # Override write as the search.json index file has already been created
+    def write(dest)
+      true
+    end
+  end
+
```
Comment (featalion) [inline _plugins/index_all_pages.rb:177]: I do not think, that it is bad situation. [Why should files end with a newline?](http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline). Is it possible to configure HoundCI to do not notify about that? See the next file, index.html, Git notifies, that there is no new line at the end of the file.

**Comment by Stephenitis** [comment]
output2

File: _plugins/index_all_pages.rb (hunk 1 above)
Comment (Stephenitis) [inline _plugins/index_all_pages.rb:177]: turning off hound.

waaay noisy. thanks but no thanks for now thoughtbot.

On Wed, Dec 17, 2014 at 3:29 PM, Yury Yantsevich notifications@github.com
wrote:

> In _plugins/index_all_pages.rb
> https://github.com/iron-io/docs/pull/414#discussion-diff-22013861:
> 
> > -      @body = @body.split.delete_if() do |x|
> > -        t = x.downcase.gsub(/[^a-z]/, '')
> > -        t.length < min_length || stopwords.include?(t)
> > -      end.join(' ')
> > -    end
> > -  end
> >   +end
> >   +module Jekyll
> >   +
> > -  class SearchIndexFile < StaticFile
> > -    # Override write as the search.json index file has already been created
> > -    def write(dest)
> > -      true
> > -    end
> > -  end
> >   +
> 
> I do not think, it is bad situation. Why should files end with a newline?
> http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline.
> Is it possible to configure HoundCI to do not notify about that? See the
> next file, index.html, Git notifies, that there is no new line at the end
> of the file.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/iron-io/docs/pull/414/files#r22013861.

## 

Stephen Nguyen,
Developer Evangelist, Iron.io http://www.iron.io/

My General Availability https://ironpairing.appointlet.com/

http://www.linkedin.com/in/stephenhnguyen/
http://www.github.com/stephenitis  http://www.twitter.com/stephenitis

**Pull request merged**